content/docs/self-hosted-runners.md

```diff
@@ -433,7 +433,95 @@ provisioned through environment variables instead of files.
 </tab>
 </toggle>
 
-### On-premise (Local) Runners
+## GitLab CI/CD and container images from private registries
+
+_See also the
+[official documentation](https://docs.gitlab.com/ee/ci/docker/using_docker_images.html#use-credential-helpers)
+for more information._
+
+### Amazon Web Services
+
+#### GitLab CI/CD Environment Variables
```

Comment:
Too much structure? Just plain sentences would work too. Or use tabs like the existing sections?